syntax = "proto3";

import "library/cpp/getoptpb/proto/confoption.proto";

message TConfig {
    option (NGetoptPb.Title) = "BertEmbedder";

    string YtProxy = 1;
    string InputTable = 2;
    string OutputTable = 3;
    string InputQueryColumnName = 4;
    string OutputEmbeddingsColumnName = 5;
    string ModelPath = 6;

    message TMapperJobParams {
        optional uint32 MemoryLimitMb = 1;
        optional uint32 DataSizePerJobMb = 2;
    }
    TMapperJobParams MapperJobParams = 7;
}
